### PR TITLE
Allow PHP 8.1 and Symfony 6.1

### LINKED ISSUE #55: Are we ready for 2.0?
FYI @chapterjason

Is there something else we should change before releasing 2.0 beta1?

### PR DESCRIPTION
**Pull request by chapterjason**
You can obtain the changes from the commits I made.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,12 +10,15 @@
...
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
-        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
-        "symfony/filesystem": "^4.4 || ^5.3 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.3 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.3 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.3 || ^6.0"
+        "php": "^7.2 || ^8.0 || ^8.1",
+        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0 || ^6.1",
```
Comment (chapterjason) [inline composer.json]: Not sure if we need to add also 5.4 to the list?

File: composer.json [hunk 2]
```diff
@@ -10,12 +10,15 @@
...
+        "symfony/filesystem": "^4.4 || ^5.3 || ^6.0 || ^6.1",
+        "symfony/framework-bundle": "^4.4 || ^5.3 || ^6.0 || ^6.1",
+        "symfony/http-kernel": "^4.4 || ^5.3 || ^6.0 || ^6.1",
+        "symfony/yaml": "^4.4 || ^5.3 || ^6.0 || ^6.1"
```
Comment (jderusse) [inline composer.json]: should be reverted `N.1` already matches `^N.0`

File: composer.json (hunk 1 above)
Comment (Nyholm) [reply composer.json]: Na, no need

**Review by Nyholm — APPROVED** [review]
Cool. Just the small change requested by @jderusse and I'm super happy with this.

File: README.md [hunk 3]
```diff
@@ -102,8 +102,8 @@ jobs:
     strategy:
       matrix:
         operating-system: [ ubuntu-latest, windows-latest ]
-        php: [ '7.4', '8.0' ]
-        symfony: ['^4.4', '^5.3']
+        php: [ '7.4', '8.0', '8.1' ]
+        symfony: ['^4.4', '^5.3', '^5.4', '^6.0', '^6.1']
```
Comment (jderusse) [inline README.md]: either `5.3.*, 5.4.*` or `~5.3.0, ~5.4.0` or `^5.3`
But you using caret in multiple minor version does not make sense: `^5.3` intersects with `^5.4`

Comment (chapterjason) [reply README.md]: @jderusse Thanks.

**Comment by chapterjason** [comment]
@Nyholm Resolved.

**Comment by Nyholm** [comment]
Awesome. Thank you

**Pull request merged**